oauth2_provider/oauth2_backends.py

```diff
@@ -180,7 +180,11 @@ def extract_body(self, request):
         :return: provided POST parameters "urlencodable"
         """
         try:
-            body = json.loads(request.body.decode('utf-8')).items()
+            data = json.loads(request.body.decode('utf-8'))
+            if isinstance(data, dict):
+                body = data.items()
+            else:
+                body = ""
```

Comment:
this body will be passed to the oauthlib for validation. this validator tries look up an access token key in the body. If we pass a list to the oauthlib we currently receive an error, because it is unable to parse the list. As a list possibly contains no access token, we can skip the body and set it to an empty string.